config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
I'm not sure it's our place to force a time format on downstream applications. At the very least, we should pick a different format name for our custom format.. but I wonder if we can just live with some of the Rails default ones?